identity/webapp/src/routes/index.ts

```diff
@@ -1,12 +1,17 @@
 import { withPrefix } from '../utility/prefix';
 import { RouteMiddleware } from '../types/application';
+import { prefix } from '../utility/prefix';
 
 const unAuthenticatedPages: string[] = ['/register', '/validated', '/error'];
 
 export const indexPage: RouteMiddleware = (context) => {
   const bundle = context.routes.url('assets-bundles');
+  let path = context.request.URL.pathname;
+  if (prefix) {
+    path = path.replace(prefix, '');
+  }
```

Comment:
Would it make more sense to leave this code as-is and modify the `unauthenticatedPages` array? Something like:

```ts
const unAuthenticatedPages: string[] = ['/register', '/validated', '/error'].map((route) => {
  return prefix ? (prefix + route) : route;
});

// ...

if (context.isAuthenticated() || unAuthenticatedPages.includes(context.request.URL.pathname)) {
```

Reply:
@ajrussellaudio I wonder if this is a time to talk about using next? (I've added it to [a list of things we will have to address](https://github.com/wellcomecollection/wellcomecollection.org/issues/6071)). I'm a little uncertain as to what part of the app we are in here? Is this is oAuth, or our wc.org application?

In terms of this implementation though - I'd probably keep the stripped path at the top of the implementation, and use that throughout the app, rather than having to remember to strip it whenever we use it elsewhere.

Reply:
@ajrussellaudio yeah, I think that's a reasonable suggestion - that way we'll not have to strip out the prefix on every request.

@jamesgorrie this is the wc.org application - this file specifically contains the base HTML for react to hook into. The customisations to Auth0 are fairly minimal and live in the [identity repo](https://github.com/wellcomecollection/identity)